File: typescript/242.valid-anagram.ts
/*
 * @lc app=leetcode id=242 lang=typescript
 *
 * [242] Valid Anagram
 */

// @lc code=start
function isAnagram(s: string, t: string): boolean {
  return s.split("").sort().join("") === t.split("").sort().join("");
}
// @lc code=end

File: python/242.valid-anagram.py
#
# @lc app=leetcode id=242 lang=python3
#
# [242] Valid Anagram
#

# @lc code=start
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        ss = sorted(s)
        tt = sorted(t)

        if(len(ss) != len(tt)):
            return False

        for i in range(len(ss)):
            if(ss[i] != tt[i]):
                return False

        return True
# @lc code=end

